appveyor.yml

```diff
@@ -17,3 +17,6 @@ cache:
 
 build: off
 skip_branch_with_pr: true
+
+init:
+- cmd: set NODE_OPTIONS=--max-old-space-size=2048
```

Comment:
I think a better way to set env var is https://www.appveyor.com/docs/build-configuration/#environment-variables.

Reply:
I'm not sure if `cmd: set` will pass the env var to next steps.